subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf5.h

```diff
@@ -9,6 +9,7 @@
 #define HAL_RADIO_NS2US_ROUND(ns) ((ns + 500)/1000)
 
 #define EVENT_TIMER NRF_TIMER0
+#define EVENT_TIMER_NBR 0
```

Comment:
Could this define also be in #ifdef guards? Same question applies to SW_SWITCH_TIMER_NBR definition.

Reply:
Why not..